include/ua_server.h

```diff
@@ -985,9 +987,21 @@ UA_Server_addMethodNode(UA_Server *server, const UA_NodeId requestedNewNodeId,
                                      outputArgumentsSize, outputArguments, UA_NODEID_NULL, NULL,
                                      nodeContext, outNewNodeId);
 }
-
 #endif
 
+#ifdef UA_ENABLE_HISTORIZING
```

Comment:
These can be all removed with the use of local MonitoredItems to update the historical data.